README.org

```diff
@@ -233,34 +233,24 @@ This is great for performance, but means the data can become stale if you modify
 The =bibtex-actions-refresh= command will reload the caches, and you can call this manually.
 You can also call any of the =bibtex-actions= commands with a prefix argument: =C-u M-x bibtex-actions-insert-key=.
 
-Although not default, =bibtex-actions= also provides convenience functions for auto-refreshing cache when bib files change using filenotify. 
-There are two such functions: =bibtex-actions-with-filenotify-local= and =bibtex-actions-filenotify-global=. 
-The following snippet will refresh caches when one of the bib files change:
+Although not default, =bibtex-actions= also provides convenience functions for auto-refreshing cache when bib files change using filenotify.
+The simplest use of this functionality is
 
 #+BEGIN_SRC emacs-lisp
-(bibtex-actions-with-filenotify-global #'bibtex-actions-refresh)
-
-(add-hook 'LaTeX-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
-(add-hook 'org-mode-hook (lambda () (bibtex-actions-with-filenotify-local #'bibtex-actions-refresh)))
+(bibtex-actions-filenotify-setup '(LaTeX-mode-hook org-mode-hook))
```

Comment:
I experimented a bit but it seems a bit tricky. Adding that line in `:config` almost works. The hook for local bibliography will not run for the first buffer because it is added too late. What works is to add it in the `:init` part. This is what I have
```lisp
(use-package! bibtex-actions
  :after latex
  :init (bibtex-actions-filenotify-setup '(LaTeX-mode-hook)))
```

Reply:
I merged it :-)

If there's some little problem, we can fix it subsequently.

Thanks!